collections/Digby/MS_Digby_185.xml

```diff
@@ -102,10 +101,209 @@
+                        <decoNote>Fol. 165r: elaborate strapwork E(xplicit).</decoNote>
+                        <decoNote type="colInit">Otherwise 1-line alternating plain red or blue initials at the beginning of stanzas; rhyme bracing in red.</decoNote>
+                        <decoNote type="drawing">Decorated catchwords, including head, fol. 135v; bird, fol. 151v, lion, fol. 159v.</decoNote>
+                     </decoDesc>
+                  </physDesc>
+                  <history>
+                     <origin>
+                        <origDate calendar="Gregorian" notAfter="1460" notBefore="1440">15th century, middle</origDate>
+                        <origPlace>
+                           <country key="place_7002445">English</country>
+                        </origPlace>
+                     </origin>
+                     <provenance>Partially erased inscription, fol. 80r: <q>T<damage><gap/> <gap/><supplied>ga</supplied>ve this</damage> <lb/> <damage><unclear>boke to T<gap/>om<gap/></unclear></damage> Clopton <lb/> and <damage><gap/></damage> of her bodye</q>; <q>W. Clopton</q>, erased (?), fol. 144v; <q>Thys boke longy<damage><gap unit="char" quantity="1"/></damage> <damage><gap/></damage> Wylly<damage><gap/></damage> Clopton</q>, erased, fol. 157r. Very probably to be identified as <persName role="fmo" key="person_3468">Thomasin, the grandaughter of Thomasin Hopton</persName> (Richmond, p. 121 n. 72), who married <persName role="fmo" key="person_3469">Sir William Clopton of Kentwell and Long Melford (d. 1531)</persName>. Section II of our manuscript may perhaps be identified with the 'book of Englissh callid Ocliff' mentioned in the testament of Thomasin the elder (d. 1498), although this was bequeathed in her testament to her son Nicholas Sidney (Richmond, p. 131).</provenance><!-- T... [gave this] / [book to (Thomasin?)] Clopton and [the heirs of her body] -->
+                    <!-- William Clopton's will is https://archive.org/details/visitationofsuff01harv/page/46/mode/2up?view=theater ; will of his son Francis checked -->
+                     <provenance>Erased names (?), illegible, fol. 165r; faint or erased names on fol. 165v, including John ... of Bokkyng [Bocking, Essex]; Thomas ?Prynce.</provenance>
+                  </history>
+               </msPart>
+               <msPart xml:id="MS_Digby_185-part3">
+                  <msIdentifier>
+                     <altIdentifier>
+                        <idno>MS. Digby 185 - codicological unit III (fols. 166-203)</idno>
```

Comment:
I am reminded of the problem with 'part' rather than 'codicological unit' after seeing someone write 'MS. Bodley 000, part X' on a request slip. It should be easy to emend the code to render `n="X"` on `<msPart>`: do you suppose that would be desirable?